toplevel/opttoploop.ml

```diff
@@ -309,7 +309,7 @@ let directive_table = (Hashtbl.create 13 : (string, directive_fun) Hashtbl.t)
 
 (* Execute a toplevel phrase *)
 
-let execute_phrase print_outcome ppf phr =
+let execute_phrase print_outcome log phr =
```

Comment:
(we could have `let ppf = Misc.Log.formatter log in` here, to make the diff smaller, as is done in `preprocess_phrase`.)